tests/unit/helpers/mocks.py

```diff
@@ -54,8 +54,9 @@ def __init__(self, *args, **kwargs):
 
 
 class MockFiles(object):
-    def __init__(self, data):
+    def __init__(self, data, mimetype='application/pdf'):
```

Comment:
I don't  like this. It's a responsibility of a `MockFileObject` to know its own default `mimetype`, not the class above it...